conseil-api/src/test/scala/tech/cryptonomic/conseil/api/config/ConseilAppConfigTest.scala

```diff
@@ -81,14 +81,14 @@ class ConseilAppConfigTest extends AnyWordSpec with Matchers with EitherValues w
                                               |    }
                                               |  },
                                               |  {
-                                              |    name: "ethereum"
+                                              |    name: "OpenChain"
```

Comment:
what does this actually change? I can't see what's impacted...

Reply:
Yea, so once we added integration for `ethereum` this test started to pass, but we expect failure here, so I changed to the blockchain we are not supporting yet ;)